android/src/main/java/io/rownd/android/Rownd.kt

```diff
@@ -401,6 +403,11 @@ class RowndClient constructor(
         signInWithGoogle(intent, hint = null)
     }
 
+    internal fun getActiveGmailAccounts(): Array<Account> {
+        val accountManager = AccountManager.get(Rownd.appHandleWrapper?.activity?.get()?.applicationContext);
```

Comment:
Probably should check for a null activity before invoking AccountManager

Reply:
Null check has been added